### ISSUE TITLE
 ask for help on dataloader.py

### ISSUE THREAD

**Issue by daixiaoxi** [leonfrank/lanenet-danet-pytorch#1]
Thanks for your coding, I learned a lot . And I want to ask for some help on  dataloader.py

          with open(file,"r") as f:
                for line in f:
                    l = line.strip("\n").split()
                    img_list.append(l[0])
                    binary_label.append(l[1])
                    instance_label.append(l[2])
            return img_list,binary_label,instance_label

 I am confused on the file contain. For each line, it contains the imgname, binary label, instance label?
each image, it should contain multi lines,the instance labels should be 1,2,3,...  so the binary label and instance label in an image  are stored in which way?
    Thanks you for advanced!

**Comment by klintan** [comment]
The train.txt file is taken from https://github.com/MaybeShewill-CV/lanenet-lane-detection/blob/master/data/training_data_example/train.txt and if you look at that each filename is delimited using a space, first image filename, then binary mask image filename and last instance label image. These are then read using cv2 later in the data loader. 

I rewrote it a tiny bit here as well https://github.com/klintan/lanenet-pytorch/blob/master/data_loader/data_loaders.py might give you some further clues.
